docs/.vitepress/config.js

```diff
@@ -1,14 +1,14 @@
-import { readFileSync } from "fs"
+import { readFileSync } from 'fs'
 import { defineConfig } from 'vitepress'
 import markdownItFootnote from 'markdown-it-footnote'
 
-// From https://github.com/ocamllabs/vscode-ocaml-platform/blob/master/syntaxes/reason.json
```

Comment:
I personally prefer to place these comments inline at the place where the file is imported than in the readme (where they are a bit out of context). But it's just my opinion, not a blocker.